data/json/items/gun/300BLK.json

```diff
@@ -114,5 +114,60 @@
     "type": "GUN",
     "name": { "str": "AR-15" },
     "default_mods": [ "retool_ar15_300blk" ]
+  },
+  {
+    "id": "sig_mcx_rattler_sbr",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "SIG MCX Rattler SBR" },
+    "description": "This is a short-barreled, gas operated rifle, with a 5.5 inch barrel, built-in thin folding stock, and full rail mount on the upper receiver.  This particular one is a select-fire military version.",
+    "weight": "2590 g",
+    "volume": "2850 ml",
+    "//": "This is the unfolded version of the weapon. Folded, it's 597 mm",
+    "longest_side": "775 mm",
+    "price": 271900,
+    "price_postapoc": 3500,
+    "color": "dark_gray",
+    "ammo": [ "300blk" ],
+    "dispersion": 160,
+    "durability": 8,
+    "min_cycle_recoil": 1700,
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 4 ] ],
+    "built_in_mods": [ "folding_stock" ],
```

Comment:
There should still be a stock modslot, so this shows correctly rather than having to scroll down to the bottom of the weapon to see it.

Reply:
I added a stock modslot, too.